### DISCUSSION TITLE
Linemode

### DISCUSSION THREAD

**Discussion by xopo** [sxyazi/yazi#1406] [1. Q&A] [answered]
I customised the linemode  using the example from here:  https://yazi-rs.github.io/docs/configuration/yazi
All good, until i sort the content or change the linemod using m. The sort functionality will use different linemode template. 

How to add a shortcut to custom when m key is pressed? Once you change to other mode I don't see how to get back to custom.

Thanks

https://github.com/user-attachments/assets/5ed5bdc0-6bdf-46b2-b2ae-437d428fa0cb

**Comment by JohannesRgnr** [comment, marked as answer]
in keymap.toml, add a line corresponding to the new mode
<img width="831" alt="Screenshot 2024-08-14 at 17 39 49" src="https://github.com/user-attachments/assets/c58558c1-d4cc-48fb-b0e3-e3c0b56facf7">

**Reply by JohannesRgnr** [reply]
or better, simply in your keymap.toml:
```

[[manager.prepend_keymap]] 
on = [ "m", "z" ] 
run = "linemode size_and_mtime" 
desc = "Set linemode to size and mtime"
```

**Reply by xopo** [reply]
Thanks, work like a charm
